local-storage-service/server/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM openjdk:11
+VOLUME /tmp
+ADD target/local-storage-service-server.jar local-storage-service-server.jar
```

Comment:
What is the difference between ADD and COPY?

Does the first path depend on where we execute build from? Better to make it independent if possible.

Reply:
It's better to use COPY command here, but we can't make the path independent. The official documentation on docker COPY command: COPY <src> <dest>.The <src> path must be inside the context of the build